mlflow/sklearn.py

```diff
@@ -89,6 +113,14 @@ def load_pyfunc(path):
     :param path: Local filesystem path to the model saved by :py:func:`mlflow.sklearn.save_model`.
     :rtype: Pyfunc format model with function
             ``model.predict(pandas DataFrame) -> pandas DataFrame``.
+
```

Comment:
Based on our discussion with @tomasatdatabricks, let's remove the example here because this method is going to be made "private" by #539.